dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -465,6 +465,46 @@ public void TipsWhichDoNotFitDoNotOpen()
             }
         }
 
+        [TestMethod]
+        public void VerifyTheming()
+        {
+            using (var setup = new TestSetupHelper("TeachingTip Tests"))
+            {
+                elements = new TeachingTipTestPageElements();
```

Comment:
>elements = new TeachingTipTestPageElements(); [](start = 16, length = 45)

I think this test should run on both TipLocationOptions, similar to the other teaching tip tests